Get message group ID correctly.

MancunianSam opened the pull request:
Lesson learned, don't just guess, read the documentation.

techncl reviewed (commented):
Looks good! Just 3 main questions:

1. Should we capture the `messageSystemAttributeNames` attribute on line 97 of the test?
2. Should we add a captor (and check the values) to each of the test for redundancy? The 2nd test has a captor but we are not verifying anything?
3. Is it worth, in one of the tests, to add a call to `client.receiveMessages` where we don't pass in a value for `maxNumberOfMessages` and test that the default is 10?

MancunianSam commented:
1. Yeah ok. I've added a check in.
2. No, I think that over-complicates things and doesn't add anything we're not already checking.
3. Yeah, good idea, I've added one in.

techncl reviewed (approved):
👍
